homeassistant/components/fitbit/sensor.py

```diff
@@ -175,9 +175,9 @@ def setup_platform(
     if "fitbit" in _CONFIGURING:
         configurator.request_done(hass, _CONFIGURING.pop("fitbit"))
 
-    access_token: str | None = config_file.get(ATTR_ACCESS_TOKEN)
-    refresh_token: str | None = config_file.get(ATTR_REFRESH_TOKEN)
-    expires_at: int | None = config_file.get(ATTR_LAST_SAVED_AT)
+    access_token = cast(str | None, config_file.get(ATTR_ACCESS_TOKEN))
+    refresh_token = cast(str | None, config_file.get(ATTR_REFRESH_TOKEN))
+    expires_at = cast(int | None, config_file.get(ATTR_LAST_SAVED_AT))
```

Comment:
Why do we need to cast now, is it because the items are no longer typed as `Any`?

Reply:
Yes

Reply:
Actually JSONValueType is ok for access_token and refresh_token.
I have now adjusted this to use walrus instead.